core/src/main/scala/spire/math/Rational.scala

```diff
@@ -879,6 +889,14 @@ private[math] object BigRationals extends Rationals[BigInt] {
 
     override def signum: Int = n.signum
 
+    override def isValidChar = false
```

Comment:
Wouldn't you need to override `isWhole` in this case as well? Very large integers should still be whole numbers.